crates/bifrost/src/read_stream.rs

```diff
@@ -434,7 +434,7 @@ fn deliver_trim_gap(
     record
 }
 
-#[cfg(test)]
+#[cfg(all(test, feature = "local-loglet"))]
```

Comment:
I am a bit concerned that we might no longer run the readstream tests if one does not specify this feature when calling `cargo nextest`. Could we try whether a self dependency where we enable this feature (https://github.com/rust-lang/cargo/issues/2911#issuecomment-1483256987) for the tests works? Alternatively, could we use the memory loglet for the tests to make them independent of the local-loglet?

Reply:
i was thinking about the self import. will give that a try